### ISSUE TITLE
When pulling data it does not match what seatgeek app shows

### ISSUE THREAD

**Issue by DallasHoelscher** [seatgeek/api-support#37]
I am currently doing a query, https://api.seatgeek.com/2/events?datetime_utc=2017-07-04&per_page=5000&client_id=SOME_CLIENT_ID I am getting events back, but not near as many as I would expect and I have found examples of missing events. 

For instance, the app shows me a local event tomorrow "https://seatgeek.com/kris-kristofferson-tickets/columbia-missouri-missouri-theatre-center-for-the-arts-2017-04-04-7-pm/concert/3600970"

This event can not be found when doing the first query, even when I change the date around to make it April 4th / 5th. I confused as to why my query won't find all events on your side that are on this exact date.

My goal is to get all events on the seatGeek side for a specific day across the United States. I appear to only get about 15 and I'd imagine thats not right. Thanks for any help you have!

**Comment by josegonzalez** [comment]
You're querying for events at *exactly* that datetime, whereas you want to find a range like so:

```
# client id missing because that should be treated as a secret on your end
https://api.seatgeek.com/2/events?datetime_utc.gte=2017-07-04&datetime_utc.lte=2017-07-05
```

Note that I'm doing `.lte` of the next day as that event you pulled is actually at midnight utc of 2017-07-05.

**Comment by DallasHoelscher** [comment]
@josegonzalez Great, Thanks! What's odd is I did indeed get tons more events, but neither of the events local to me showed up in the list. I also went on the app and found some events in St. Louis, for example "Chicago Cubs at St. Louis Cardinals", even when I query between "datetime_utc.gte=2017-07-03&datetime_utc.lte=2017-07-06" I actually do not see any of the events the SeatGeek app shows me in St. Louis. Is there something simple I'm missing?
